pandas/tests/groupby/test_function.py

```diff
@@ -585,6 +585,36 @@ def test_ops_general(op, targop):
     tm.assert_frame_equal(result, expected)
 
 
+@pytest.mark.parametrize(
+    "op",
```

Comment:
There should already be a fixture for this called `reduction_func` you can use

Reply:
Thanks - I tried it out and there are some that fail. Not sure how to incorporate corrwith and nth - perhaps these two should be skipped. idxmax, idxmin, mad, nunique, and skew all modify column "a" in the test, and ngroup and size don't include "a" in the output when as_index=False. It seems to me that these are not desired behaviors should be fixed. I'll take a look and see if I can do so.

Under the assumption that I can't, is it better to leave the list as is, or replace with the fixture and skip these?

Reply:
After looking into this some more, I found I'm running into #5755. It seems to me more agreement is needed before deciding a way forward for these. I've updated the test using the fixture and skipping these for now.